eventually-core/src/aggregate.rs

```diff
@@ -56,14 +56,12 @@ pub trait Aggregate {
     /// of [`Event`](Aggregate::Event)s to apply the
     /// [`State`](Aggregate::State) mutation based on the current representation
     /// of the State.
-    fn handle<'a, 's: 'a>(
+    fn handle<'a>(
```

Comment:
Do we even need to specify the lifetime if it's only one, or elision is smart enough to figure that out?

Reply:
i like where your head's at! actually i took this a step further and used `async-trait` to define it.